Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WMRDevice.cs

```diff
@@ -35,6 +40,11 @@ public void Initialize()
 
         private void InitializeSources()
         {
+            //Tried but it fails to build and causes errors in Unity :S (recognized in the Player project)
```

Comment:
Am i reading correctly that this compiles in VS but not in the Unity Editor?

Reply:
Most likely from an open bug I have reported about the custom assembly definitions